### PR TITLE
Initial build

### PR DESCRIPTION
**Pull request by johnhughes1984**
See README for further detail/context

### CODE REVIEW

File: Plugin/OverridePageBuilderTemplates.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+     * See: vendor/magento/module-page-builder/view/adminhtml/web/js/content-type/products/mass-converter
+     *
+     * @param ProductsList $subject
+     * @param string|null $result
+     *
+     * @return string|null
+     */
+    public function afterGetTemplate(ProductsList $subject, ?string $result): ?string
+    {
+        if (strpos($result, self::GRID_WIDGET_TEMPLATE) !== false) {
```
Comment (pykettk) [inline Plugin/OverridePageBuilderTemplates.php:26]: Do we care about case sensitivity here?

File: view/frontend/templates/catalog/product/widget/content/carousel.phtml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    $items = $block->getProductCollection()->getItems();
+    $templateType = ReviewRendererInterface::SHORT_VIEW;
+    $description = false;
+    ?>
+    <div class="glider-contain product-items <?= /* @noEscape */ $type ?>">
+        <div data-role="glider-pagebuilder-product"
+             class="mx-auto grid gap-4 sm:grid-cols-2 lg:grid-cols-4 relative overflow-y-hidden overflow-x-scroll">
+            <?php /** @var Product $product */ ?>
+            <?php foreach ($items as $item): ?>
+                <?= $block->getLayout()->getBlock('product_list_item')
```
Comment (pykettk) [inline view/frontend/templates/catalog/product/widget/content/carousel.phtml:31]:
```suggestion
                <?= /** @noEscape */ $block->getLayout()->getBlock('product_list_item')
```

File: view/frontend/templates/catalog/product/widget/content/carousel.phtml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                    ->setData('product', $item)
+                    ->setData('view_mode', $mode)
+                    ->setData('image_display_area', $image)
+                    ->setData('show_description', $description)
+                    ->setData('template_type', $templateType)
+                    ->toHtml() ?>
+            <?php endforeach ?>
+        </div>
+        <?= $block->getBlockHtml('pagebuilder.carousel.nav') ?>
```
Comment (pykettk) [inline view/frontend/templates/catalog/product/widget/content/carousel.phtml:40]:
```suggestion
        <?= /** @noEscape */ $block->getBlockHtml('pagebuilder.carousel.nav') ?>
```

File: view/frontend/templates/catalog/product/widget/content/grid.phtml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        <?php if ($block->getTitle()): ?>
+            <div class="block-title">
+                <strong><?= $escaper->escapeHtml(__($block->getTitle())) ?></strong>
+            </div>
+        <?php endif ?>
+        <div class="block-content">
+            <div class="mx-auto grid gap-4 <?= $mode === 'grid' ? 'sm:grid-cols-2 lg:grid-cols-4' : 'grid-cols-1' ?>">
+                <?php /** @var Product $product */ ?>
+                <?php foreach ($items as $item): ?>
+                    <?= $block->getLayout()->getBlock('product_list_item')
```
Comment (pykettk) [inline view/frontend/templates/catalog/product/widget/content/grid.phtml:38]:
```suggestion
                    <?= /** @noEscape */ $block->getLayout()->getBlock('product_list_item')
```

File: view/frontend/templates/catalog/product/widget/content/grid.phtml [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+                        ->setData('product', $item)
+                        ->setData('view_mode', $mode)
+                        ->setData('image_display_area', $image)
+                        ->setData('show_description', $description)
+                        ->setData('template_type', $templateType)
+                        ->toHtml() ?>
+                <?php endforeach ?>
+            </div>
+            <?= $block->getPagerHtml() ?>
```
Comment (pykettk) [inline view/frontend/templates/catalog/product/widget/content/grid.phtml:47]:
```suggestion
            <?= /** @noEscape */ $block->getPagerHtml() ?>
```

**Review by pykettk — CHANGES_REQUESTED** [review]
Just minor documentation changes from me.

File: Plugin/OverridePageBuilderTemplates.php (hunk 1 above)
Comment (johnhughes1984) [reply Plugin/OverridePageBuilderTemplates.php:26]: In theory* you could have modules with the same name in different cases, so we should keep the case, but appreciated that scenario will likely never arise and cause its own set of problems! I think my answer would be it probably doesn't matter too much either way.

> *Never tried in practice to confirm if it works

**Review by pykettk — DISMISSED** [review]

**Review by rich-humphreys — DISMISSED** [review]

**Review by amanda107 — APPROVED** [review]

**Review by pykettk — DISMISSED** [review]

**Review by rob-aimes — APPROVED** [review]

**Review by amanda107 — APPROVED** [review]

**Review by fredden — APPROVED** [review]

**Pull request merged**